packages/tables/src/Concerns/HasBulkActions.php

```diff
@@ -188,4 +188,9 @@ protected function getTableBulkActions(): array
     {
         return [];
     }
+
+    protected function getTableRecordBulkActionsEnabled(): ?Closure
```

Comment:
This shouldn't be `protected` if we need to access it from outside the class.

Reply:
Agree, changed.